circleci/images/pgupgradetester/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# FROM python:3.6-slim
+FROM python@sha256:6411a78d12cd3bd1e594c3dd1532ed02e62da744bbc92863a7548ae2de9e7dfa
```

Comment:
most of our images use the FROM statement without the hash, but the tag.
benefit is that when we rebuild it might pull in (security) fixes that have been made upstream. Downside is that builds might be less reproducible, although I expect it to be pretty reporducible with python 3.6 and our pip file.

Any reasoning why you went for the hash pinned one here?
If there is no strong reason I would suggest going with `FROM python:3.6-slim`

Reply:
There is no strong reason so I will update this.